packages/core-modules/contracts/modules/CoreElectionModule.sol

```diff
@@ -214,22 +214,17 @@ contract CoreElectionModule is IElectionModule, ElectionStorage, OwnableMixin {
         return !isNominating() && !isEpochFinished();
     }
 
-    function setupFirstEpoch() external virtual override {
+    function setupFirstEpoch() external override onlyOwner {
         ElectionStore storage store = _electionStore();
 
         if (store.epochStart != 0) {
             revert FirstEpochAlreadySetUp();
         }
 
-        // TODO set epoch 0 seat to 1
-        // TODO set epoch 0 period
-        // TODO set token for owner
-
-        // Flip epochs
-        // solhint-disable-next-line not-rely-on-time
-        store.epochStart = block.timestamp;
-        store.seatCount = store.nextSeatCount;
-        store.epochDuration = store.nextEpochDuration;
-        store.nominationPeriodPercent = store.nextNominationPeriodPercent;
+        store.epochStart = block.timestamp; // solhint-disable-line not-rely-on-time
+        store.seatCount = 1;
+        store.epochDuration = 2 days;
+        store.nominationPeriodPercent = 0;
+        // TODO: Assign current owner as only council member
```

Comment:
Is this a good idea, to start with a council of just the owner?

Reply:
I think so. It's better to have 1 head than to be headless right?

Reply:
In fact this "owner" will be the setup owner or pDAO setting up the module. But we should be careful if the owner is the council itself (and there's no council defined)